channelmonitor/channelmonitor.go

```diff
@@ -382,6 +384,9 @@ func (mc *monitoredChannel) restartChannel() {
 
 		if !restartAgain {
 			// No restart queued, we're done
+			if mc.cfg.OnRestartComplete != nil {
+				mc.cfg.OnRestartComplete(mc.chid)
```

Comment:
I don't see this getting configured/injected anywhere. What's the purpose of this ?

Reply:
This is just used by the tests